governance/paper-process.md

```diff
@@ -0,0 +1,235 @@
+of the authors to capture the different views with references, benefits, and
+drawbacks or contingencies in order to provide the reader with enough
+information to reach their own conclusion.
+
+#### References
+
+It is important that any references used to generate paper content be cited
+appropriately through footnotes and within the references section.  This
+permits the reader to not only dig deep into a specific reference area but also
+provides source content and is a matter of good practice.  
+
+### Collaborative
+
+Contributors now review all content roughed-in and transition raw content into
+draft content.  They may begin commenting on sections, expanding ideas, refining
+arguments, and providing citations.  Contributors are encouraged to discuss
+content specific items in the doc through comments whereas larger themes and
+writing decisions should be discussed in the corresponding slack channel.  This
+keeps the doc clean as well as documents thematic decisions in the primary
+collaboration & coordination channel.
+
+### Executive summary and content wrap-up
+
+If the paper exceeds five (5) pages, an Executive summary commensurate with the
+length of the document (but not to exceed 3 pages) should be written.  Content,
+comments, and other discussions should be closed out and finalized at this time.
+
+### Narrative voice
+
+No more than three contributing individuals should be assigned for narrative
+voice. The narrative voice is a semi-final pass of the paper to ensure it reads
+as a single, unified voice. It should ensure:
+
+* the language origin is consistent throughout the document (lang_en or
+  lang_us),
+* phrasing is similar (caddy corner not mixed in with kitty corner),
+* acronyms are spelled out at their first use and then abbreviated later,
+* footnotes and citations are consistent and not direct hyperlinks in the text
+* vague terms are defined in a glossary or otherwise cited to the cloud native
+  environmental sustainability lexicon in the repo
+
+### Final group review
+
+This is the opportunity for the group to collectively review the polished paper
+prior to opening it for community review.  This should resolve and close out any
+outstanding items.
+
+### Community Review
+
+At this point the paper needs to be locked from editing with the exception of
+the Adjudicators (see below).  It should be made public with the permission
+settings such that suggestions and comments are permitted.  The lead will then
+provide a brief write up of a call to action with a link to the document and the
+due by date for comments.  This write up and corresponding links and details
+will be emailed to the Environmental Sustainability WG mailing list, at which point we
+actively solicit public comment.
+
+### Public comment adjudication
+
+Prior to opening the paper for community review, the group should select no more
+than 3 Adjudicators.  These individuals are tasked with accepting/rejecting
```

Comment:
why 3?

Reply:
The more you have in the discussion the longer you will need to find a consensus.

Reply:
The community will reach the consensus through public comment and deliberation. Vivid discussions are encouraged, not limited. Setting an arbitrary upper bound is not the way the CNCF community operates.